documentation/optee_design.md

```diff
@@ -315,9 +315,105 @@ TA. This is solved by always setting the TA context in the CPU when
 resuming execution. Here's room for improvements since it's more likely
 than not that it's the same CPU that resumes execution in secure world.
 
-# 6. SVC
-Will be written soon.
-
+# 6. Stacks
+Different stacks are used during different stages. The stacks are:
+- Secure monitor stack (128 bytes), bound to the CPU. Only available if
+  OP-TEE is compiled with a secure monitor always the case if the target is
+  ARMv7 but never for ARMv8.
```

Comment:
Yes, but it has some drawbacks also. If we say ARMv7-A then we're precise and at the same time excludes ARMv7-M or whatever. Everything we're currently doing we're doing on -A processors, but is everything only applicable for -A processors? 

What should I replace `ARMv7/AArch32` with?